content/App.tsx

```diff
@@ -35,6 +35,29 @@ const queryClient = new QueryClient({
   },
 });
 
+export const stakingTheme: Partial<ChakraTheme> = {
+  ...chakraTheme,
+  components: {
+    Button: {
+      variants: {
+        solid: (_props) => ({
+          _hover: {
+            bgGradient: chakraTheme.gradients.cyan,
```

Comment:
I think we can just use like navy.900 (or whatever the shortcut is for gradient) here, pretty sure we don't need the theme object.

Reply:
This is just added to test that global theme alteration gets applied to the storybook.  Currently it does not :(